app/collections/settings.jsx

```diff
@@ -84,6 +89,37 @@ export default class CollectionSettings extends React.Component {
       });
   }
 
+  handleDescriptionInputChange() {
+    const property = 'description';
```

Comment:
Add a name attribute to the textarea and then you can use the event object: `event.target.name`